app/views/hyrax/dashboard/_collections.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="row">
+  <div class="col-md-12">
+    <div id="accordion">
+      <div class="panel panel-default">
+
+        <div class="panel-heading">
+          <h2 class="panel-title"><%= t('.title') %></h2>
+          <div><%= t('.recent_activity') %></div>
+        </div>
+
+        <div class="panel-body">
+          <table class="table table-striped">
+            <thead>
+              <tr>
+                <th><%= t('.collection_title') %></th>
+                <th><%= t('.last_updated') %></th>
+                <th><%= t('.works') %></th>
+                <th><%= t('.files') %></th>
+                <th></th>
+              </tr>
+            </thead>
+            <tbody>
+             <% @collections.each do |collection| %>
+                <tr>
+                  <td><%= collection %></td>
+                  <td><%= collection.modified_date.strftime("%Y-%m-%d") %></td>
+                  <td><%= collection.member_works.count %></td>
+                  <td><%= collection.total_file_sets %></td>
+                  <td>
+                    <% pinned = collection.pinned?(current_user.id) %>
```

Comment:
i'd prefer this as a helper instead of a collection method.

we're trying in general to avoid introducing new dependencies on model layer specifics to the views.

```ruby
def pinned_collection?(collection:, user:)
  UserPinnedCollection.where(user_id: user_id, collection_id: id).exists?
end
```

otherwise i'd be okay making this a class method on `UserPinnedCollection`, like `UserPinnedCollection.pinned?(collection_id:, user_id:)` ?

Reply:
Removed for now (part of pinned collections feature).